__tests__/api.test.js

```diff
@@ -309,3 +309,66 @@ describe("POST /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("PATCH /api/articles/:article_id", () => {
+  it("PATCH status: 200, responds with an updated article, with incremented votes by given number ", () => {
+    const newVotes = { inc_votes: 10 };
+    return request(app)
+      .patch("/api/articles/5")
+      .send(newVotes)
+      .then((response) => {
+        expect(response.status).toBe(200);
```

Comment:
It would be better if we checked the data types of the keys. Checks the presence of values and the obviously the data type.